RedHatInsights/insights-inventory-frontend
# fix(ESSNTL-5239): Improve RBAC check for bulk group actions

Implements https://issues.redhat.com/browse/ESSNTL-5239.

Must be merged after https://github.com/RedHatInsights/insights-inventory-frontend/pull/1973 and https://github.com/RedHatInsights/frontend-components/pull/1885.

This makes sure that both **bulk** actions (Remove from group and Add to group) on the /inventory page comply with the RBAC requirements: specifically that the table no longer checks on the global level permissions.

- [x] Write cy tests

## How to test

1. Set up `inventory:groups:write` permission only to specific group
2. Navigate to /inventory
3. Select one or more ungroupped hosts and make sure "Add to group" is enabled; try to add the groups 
4. Select some systems in a group that you don't have write permissions: you shouldn't be able to bulk remove them from the group
5. Select some systems in a group that you have write permissions: you should be able to remove them via bulk Remove from group 
6. Select the same systems as in 5. + some ungroupped hosts: you still should be able to remove the hosts that are in the group, and the modal should correctly display the number of hosts to be removed.
